src/lib.rs

```diff
@@ -53,26 +53,37 @@
 //! and run the main event loop:
 //!
 //! ```no_run
-//! # extern crate gtk;
-//! # use gtk::prelude::*;
-//! # use gtk::{Window, WindowType};
+//! extern crate gtk;
+//! extern crate gio;
+//!
+//! // To import all needed traits.
+//! use gtk::prelude::*;
+//! use gio::prelude::*;
+//!
+//! use std::env;
+//!
 //! fn main() {
-//!     gtk::init().unwrap();
-//!     // Create the main window.
-//!     let window = Window::new(WindowType::Toplevel);
-//!     // UI initialization.
-//!     // ...
-//!     // Don't forget to make all widgets visible.
-//!     window.show_all();
-//!     // Handle closing of the window.
-//!     window.connect_delete_event(|_, _| {
-//!         // Stop the main loop.
-//!         gtk::main_quit();
-//!         // Let the default handler destroy the window.
-//!         Inhibit(false)
+//!     let uiapp = gtk::Application::new("org.gtkrsnotes.demo",
+//!                                       gio::ApplicationFlags::FLAGS_NONE)
+//!                                  .expect("Application::new failed");
+//!     uiapp.connect_activate(|app| {
```

Comment:
Strange, UI builds in `connect_startup` and `connect_activate` empty in examples,
but gtk examples https://gitlab.gnome.org/GNOME/gtk/blob/master/examples/bp/bloatpad.c shows that UI needed be created in `connect_activate`,
seems we need change examples.

Reply:
Ah indeed!